datadog/resource_datadog_user.go

```diff
@@ -2,24 +2,13 @@ package datadog
 
 import (
 	"context"
-	"log"
-	"regexp"
-	"strings"
-
-	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
-
 	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/zorkian/go-datadog-api"
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+	"log"
```

Comment:
```suggestion
```

Reply:
The log import is used in the resource to print some helpful messages.